test/ng/directive/validatorsSpec.js

```diff
@@ -730,5 +730,20 @@ describe('validators', function() {
 
       expect(helper.validationCounter.required).toBe(1);
     });
+
+    it('should validate once when inside ngRepeat, and set the "required" error when ngRequired is false by default', function() {
+      $rootScope.isRequired = false;
+      $rootScope.refs = {};
+
+      var elm = helper.compileInput(
+        '<div ng-repeat="input in [0]">' +
+        '<input type="text" ng-ref="refs.input" ng-ref-read="ngModel" ng-model="value" ng-required="isRequired" validation-spy="required" />'
+        + '</div>'
```

Comment:
The `+` sign placement seems weird (i.e. inconsistent - which is the ultimate form of weird 😛).
Did the linter have you do it like this, or is it a personal preference?

Reply:
Ha, this happened because I tested without ngRepeat, and switching between with / without is easier if the concat plusses are at the start of the line